x-pack/plugins/lens/public/editor_frame_service/editor_frame/suggestion_panel.scss

```diff
@@ -27,13 +28,26 @@
   margin-left: $euiSizeXS / 2;
   margin-bottom: $euiSizeXS / 2;
 
+  &:focus {
+    @include euiFocusRing;
+    transform: none !important; // sass-lint:disable-line no-important
+  }
+
   .lnsSuggestionPanel__expressionRenderer {
     position: static; // Let the progress indicator position itself against the button
   }
 }
 
 .lnsSuggestionPanel__button-isSelected {
```

Comment:
Please also add the following nested styles within this selector's styles. Doing so will prevent the selected button from transitioning up 2px on hover (which feels odd on the selected button).

```
&:hover {
  transform: none !important; // sass-lint:disable-line no-important
}
```